src/components/_basic.less

```diff
@@ -35,6 +35,7 @@ a.sbtn {
   &:hover,
```

Comment:
remove `&:hover, &:active` it should be on `a.sbtn` and it'll be applied for all states as well.

Reply:
ok